install/Knative-with-ICP.md

```diff
@@ -0,0 +1,139 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires a [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v2.1.0.3 or newer. The install step you can find on the IBM Knowledge Center, [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html)
```

Comment:
s/a/an

Reply:
You are mentioning `cluster v2.1.0.3`, but here you are using 3.1.1 document, so here I'd like you mention this example is using 3.1.1 as example.